packages/fx-core/src/core/index.ts

```diff
@@ -524,7 +524,7 @@ export class FxCore implements Core {
             description: "",
             author: "",
             scripts: {
-              test: "echo \"Error: no test specified\" && exit 1",
+              test: 'echo "Error: no test specified" && exit 1',
```

Comment:
Better to not change others' codes if not necessary for review simplicity.

Reply:
it's commit hook, out of my control.